src/SmartDetectorsApplianceShared/SmartDetectorRepository.cs

```diff
@@ -196,6 +198,13 @@ private static SmartDetectorManifest GenerateSmartDetectorManifest(CloudBlob sma
                     .ToList();
             }
 
+            List<DetectorParameterDefinition> parametersDefinitions = null;
+            if (smartDetectorMetadata.ContainsKey("parametersDefinitions"))
+            {
+                parametersDefinitions =
+                    JsonConvert.DeserializeObject<List<DetectorParameterDefinition>>(smartDetectorMetadata["parametersDefinitions"]);
```

Comment:
>parametersDefinitions [](start = 108, length = 21)

Please update the ps script that uploads the package

Reply:
there's no need - you wrote it too good!

---
In reply to: [233436046](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/23#discussion_r233436046) [](ancestors = 233436046)